tardis/io/logger/logger.py

```diff
@@ -1,8 +1,8 @@
 import logging
 import sys
-import logging
 import warnings
 import pyne.data
+
```

Comment:
why a newline?

Reply:
Just to demarcate the different levels of imports. Native library imports, imports from external modules, etc